tests/console/journalctl.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test basic journalctl functionality
+# Maintainer: Sergio Lindo Mansilla <slindomansilla@suse.com>
+# Tags: poo#27050
```

Comment:
You can also reference bsc#1063066 here which might be even more helpful as a progress ticket to give the motivation for the test module.